cli/azd/pkg/auth/manager.go

```diff
@@ -55,6 +55,9 @@ const cUseCloudShellAuthEnvVar = "AZD_IN_CLOUDSHELL"
 // The scopes to request when acquiring our token during the login flow or when requesting a token to validate if the client
 // is logged in.
 var LoginScopes = []string{azure.ManagementScope}
+var loginScopesMap = map[string]struct{}{
```

Comment:
We might consider building the value here (from LoginScopes) in an `init()` function for this package if we ever get to a point where we have multiple scopes (and don't want to maintain the list in two logical places).